packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -4234,7 +4234,7 @@ module.exports = function(config, DB) {
 
       beforeEach(async () => {
         account = createAccount();
-        return db.createAccount(account.uid, account);
+        return db.createAccount(account.uid, account).delay(100);
```

Comment:
Rather than add the delay for every test case in this set, can you move it into the ones that actually need it? I count 19 test cases below, so that's nearly 2 seconds of delay added in total.

And probably doesn't need to be as much as 100 milliseconds does it? (although less of an issue if we're only doing it twice of course)